### ISSUE TITLE
Cluster Autoscaler crashing in EKS China region cn-north-1- version 1.19.1

### ISSUE THREAD

**Issue by marcincuber** [kubernetes/autoscaler#3892]
**Which component are you using?**:

cluster-autoscaler

**What version of the component are you using?**:

Component version:

**What k8s version are you using (`kubectl version`)?**:
```
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.4", GitCommit:"d360454c9bcd1634cf4cc52d1867af5491dc9c5f", GitTreeState:"clean", BuildDate:"2020-11-14T14:49:35Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"19+", GitVersion:"v1.19.6-eks-49a6c0", GitCommit:"49a6c0bf091506e7bafcdb1b142351b69363355a", GitTreeState:"clean", BuildDate:"2020-12-23T22:10:21Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"linux/amd64"}
```

**What environment is this in?**:

EKS China cn-north-1

**What did you expect to happen?**:

**What happened instead?**:

crashes with the following:

```
Trace[605764324]: [99.363005ms] [99.363005ms] END
W0219 15:43:53.693599       1 aws_util.go:86] Error unmarshalling https://pricing.us-east-1.amazonaws.com/offers/v1.0/aws/AmazonEC2/current/cn-north-1/index.json, skip...
F0219 15:43:53.693635       1 aws_cloud_provider.go:358] Failed to generate AWS EC2 Instance Types: unable to load EC2 Instance Type list
goroutine 61 [running]:
k8s.io/autoscaler/cluster-autoscaler/vendor/k8s.io/klog/v2.stacks(0xc00000e001, 0xc000fd0000, 0x8a, 0xfd)
	/gopath/src/k8s.io/autoscaler/cluster-autoscaler/vendor/k8s.io/klog/v2/klog.go:996 +0xb8
```

**Anything else we need to know?**:

I tried providing region variables but that didn't help.
```
            - name: AWS_REGION
              value: cn-north-1
            - name: AWS_DEFAULT_REGION
              value: cn-north-1
```

Any advise how to make cluster autoscaler work in `cn-north-1` region in EKS?

**Comment by marcincuber** [comment]
I just built a cluster-autoscaler image by running `make make-image` from master branch. Good news is that whatever is on master actually works in china and there are no issues with autoscaler. So I guess we are just missing a release.

**Comment by yongzhang** [comment]
You can build an image from branch `cluster-autoscaler-release-1.19`, see https://github.com/kubernetes/autoscaler/pull/3697/files,  is wasn't merged into 1.19.1 release.
